oem/ibm/requester/dbus_to_file_handler.cpp

```diff
@@ -137,12 +145,18 @@ void DbusToFileHandler::processNewResourceDump(
 
     // Need to reconsider this logic to set the value as "1" when we have the
     // support to handle multiple resource dumps
-    fs::path resDumpFilePath = resDumpDirPath / "1";
 
-    std::ofstream fileHandle;
-    fileHandle.open(resDumpFilePath, std::ios::out | std::ofstream::binary);
+    std::string pathString = static_cast<std::string>(resDumpCurrentObjPath);
+    std::string filename = fs::path(pathString).filename().string();
+
+    info("In processNewResourceDump fileHandle is {FILEHANDLE}", "FILEHANDLE",
```

Comment:
Pls change FILEHANDLE to match with filename

Reply:
done